spec/models/organization_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe Organization, type: :model do
+  context 'validation tests' do
+    it 'ensures name presence' do
+      organization = Organization.new(name: 'Sample').save
+      expect(organization).to eq(true)
```

Comment:
please check `organization .valid?` , one more - add test where record is invalid